src/_content/schedule/talks/2025-09-10-09-15-t0-keynote-to-be-announced-wednesday.md

```diff
@@ -3,9 +3,14 @@ category: talks
 difficulty: All
 end_datetime: 2025-09-10 10:00:00-05:00
 permalink: /talks/keynote-wednesday/
-presenter_slugs: null
+presenter_slugs: 
+- marlene
 room: Room A
 start_datetime: 2025-09-10 09:15:00-05:00
-title: Keynote (to be announced) (Wednesday)
+title: Django Reimagined For The Age of AI 
 track: t0
 ---
+22 years ago Django was created in a newsroom by two programmers, Simon Wilison and Adrian Holovaty. Since then, it's been downloaded millions of times and has one of the most active open source communities in Python. At the same time the software industry has changed a lot because of AI, leaving many of us feeling unsure about the future of web development. In this talk we'll look at some of the key ideas that made Django successful in its early days, and how we can apply them today to reimagine how we create web applications with AI.
```

Comment:
```suggestion
22 years ago Django, was created in a newsroom by two programmers, Simon Willison and Adrian Holovaty. Since then, it's been downloaded millions of times and has one of the most active open source communities in Python. At the same time, the software industry has changed a lot because of AI, leaving many of us feeling unsure about the future of web development. In this talk we'll look at some of the key ideas that made Django successful in its early days, and how we can apply them today to reimagine how we create web applications with AI.
```